troi/tests/test_utils.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from troi.utils import discover_patches
+from troi.patch import Patch
+
+
+class TestPatches(unittest.TestCase):
+
+    def test_discover_patches(self):
+        patches = discover_patches()
+
+        assert len(patches) == 2
+        assert "daily-jams" in patches
+        assert "area-random-recordings" in patches
+
+        print(patches)
```

Comment:
extra print

Reply:
Fixed and committed to main, I didn't see these comments at first.

Reply:
not sure if you prefer that I mark reviews as changes requested even when there are only small changes to be made? I didn't think that it was worth doing 2 rounds for such a small change. Or I can be more explicit on the review message.

Reply:
I think if you used "request changes" if there are issues that you really want to see fixed and "approve" if you're fine with merging, regardless of what comments you've left, I think that would work well for me. WDYT?